pip install chatterbot error

i am using python 3.8 64 bit and trying to install chatterbot using the command:
pip install chatterbot

but it shows following error:

C:\Users\user>pip install chatterbot
Collecting chatterbot
  Using cached https://files.pythonhosted.org/packages/6c/0e/dac0d82f34f86bf509cf5ef3e2dfc5aa7d444bd843a2330ceb7d854f84f2/ChatterBot-1.0.5-py2.py3-none-any.whl
Collecting spacy<2.2,>=2.1
  Using cached https://files.pythonhosted.org/packages/1f/e2/46650d03c7ff2b57ed7af211d41c3f606540f7adea92b5af65fcf9f605c0/spacy-2.1.9.tar.gz
  Installing build dependencies ... error
  ERROR: Command errored out with exit status 1:
   command: 'c:\users\user\appdata\local\programs\python\python38\python.exe' 'c:\users\user\appdata\local\programs\python\python38\lib\site-packages\pip' install --ignore-installed --no-user --prefix 'C:\Users\user\AppData\Local\Temp\pip-build-env-qwm0zsrq\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools 'wheel>0.32.0,<0.33.0' Cython 'cymem>=2.0.2,<2.1.0' 'preshed>=2.0.1,<2.1.0' 'murmurhash>=0.28.0,<1.1.0' 'thinc>=7.0.8,<7.1.0'
       cwd: None
  Complete output (193 lines):
  Collecting setuptools
    Using cached https://files.pythonhosted.org/packages/d9/de/554b6310ac87c5b921bc45634b07b11394fe63bc4cb5176f5240addf18ab/setuptools-41.6.0-py2.py3-none-any.whl
  Collecting wheel<0.33.0,>0.32.0
    Using cached https://files.pythonhosted.org/packages/ff/47/1dfa4795e24fd6f93d5d58602dd716c3f101cfd5a77cd9acbe519b44a0a9/wheel-0.32.3-py2.py3-none-any.whl
  Collecting Cython
    Using cached https://files.pythonhosted.org/packages/41/2c/9d873fc8d1be29af12a1d41707461399327396da10e50e183754aa4136b9/Cython-0.29.14-cp38-cp38-win_amd64.whl
  Collecting cymem<2.1.0,>=2.0.2
    Using cached https://files.pythonhosted.org/packages/8b/dc/0976e04cc46f86e0dd3ee3797ec68057eaafebf31daca9a076dc138b9920/cymem-2.0.2.tar.gz
  Collecting preshed<2.1.0,>=2.0.1
    Using cached https://files.pythonhosted.org/packages/0b/14/c9aa735cb9c131545fc9e23031baccb87041ac9215b3d75f99e3cf18f6a3/preshed-2.0.1.tar.gz
  Collecting murmurhash<1.1.0,>=0.28.0
    Using cached https://files.pythonhosted.org/packages/22/e9/411be1845f1ac07ae3bc40a4b19ba401819baed4fa63b4f5ef28b2300eb4/murmurhash-1.0.2.tar.gz
  Collecting thinc<7.1.0,>=7.0.8
    Using cached https://files.pythonhosted.org/packages/92/39/ea2a3d5b87fd52fc865fd1ceb7b91dca1f85e227d53e7a086d260f6bcb93/thinc-7.0.8.tar.gz
      ERROR: Command errored out with exit status 1:
       command: 'c:\users\user\appdata\local\programs\python\python38\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-d360k4uy\\thinc\\setup.py'"'"'; __file__='"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-d360k4uy\\thinc\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\user\AppData\Local\Temp\pip-install-d360k4uy\thinc\pip-egg-info'
           cwd: C:\Users\user\AppData\Local\Temp\pip-install-d360k4uy\thinc\
      Complete output (173 lines):
      Could not locate executable g77
      Could not locate executable f77
      Could not locate executable ifort
      Could not locate executable ifl
      Could not locate executable f90
      Could not locate executable DF
      Could not locate executable efl
      Could not locate executable gfortran
      Could not locate executable f95
      Could not locate executable g95
      Could not locate executable efort
      Could not locate executable efc
      Could not locate executable flang
      don't know how to compile Fortran code on platform 'nt'
      'svnversion' is not recognized as an internal or external command,
      operable program or batch file.
      non-existing path in 'numpy\\distutils': 'site.cfg'
      Running from numpy source directory.
      C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\setup.py:418: UserWarning: Unrecognized setuptools command, proceeding with generating Cython sources and expanding templates
        run_build = parse_setuppy_commands()
      C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\system_info.py:690: UserWarning:
          Optimized (vendor) Blas libraries are not found.
          Falls back to netlib Blas library which has worse performance.
          A better performance should be easily gained by switching
          Blas library.
        self.calc_info()
      C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\system_info.py:690: UserWarning:
          Blas (http://www.netlib.org/blas/) libraries not found.
          Directories to search for the libraries can be specified in the
          numpy/distutils/site.cfg file (section [blas]) or by setting
          the BLAS environment variable.
        self.calc_info()
      C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\system_info.py:690: UserWarning:
          Blas (http://www.netlib.org/blas/) sources not found.
          Directories to search for the sources can be specified in the
          numpy/distutils/site.cfg file (section [blas_src]) or by setting
          the BLAS_SRC environment variable.
        self.calc_info()
      C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\system_info.py:1712: UserWarning:
          Lapack (http://www.netlib.org/lapack/) libraries not found.
          Directories to search for the libraries can be specified in the
          numpy/distutils/site.cfg file (section [lapack]) or by setting
          the LAPACK environment variable.
        if getattr(self, '_calc_info_{}'.format(lapack))():
      C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\system_info.py:1712: UserWarning:
          Lapack (http://www.netlib.org/lapack/) sources not found.
          Directories to search for the sources can be specified in the
          numpy/distutils/site.cfg file (section [lapack_src]) or by setting
          the LAPACK_SRC environment variable.
        if getattr(self, '_calc_info_{}'.format(lapack))():
      c:\users\user\appdata\local\programs\python\python38\lib\distutils\dist.py:274: UserWarning: Unknown distribution option: 'define_macros'
        warnings.warn(msg)
      Traceback (most recent call last):
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\sandbox.py", line 154, in save_modules
          yield saved
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\sandbox.py", line 195, in setup_context
          yield
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\sandbox.py", line 250, in run_setup
          _execfile(setup_script, ns)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\sandbox.py", line 45, in _execfile
          exec(code, globals, locals)
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\setup.py", line 443, in <module>
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\setup.py", line 435, in setup_package
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\core.py", line 171, in setup
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\__init__.py", line 145, in setup
          return distutils.core.setup(**attrs)
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\core.py", line 148, in setup
          dist.run_commands()
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\dist.py", line 966, in run_commands
          self.run_command(cmd)
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\dist.py", line 985, in run_command
          cmd_obj.run()
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\command\bdist_egg.py", line 163, in run
          self.run_command("egg_info")
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\cmd.py", line 313, in run_command
          self.distribution.run_command(command)
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\dist.py", line 985, in run_command
          cmd_obj.run()
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\command\egg_info.py", line 26, in run
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\cmd.py", line 313, in run_command
          self.distribution.run_command(command)
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\dist.py", line 985, in run_command
          cmd_obj.run()
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\command\build_src.py", line 142, in run
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\command\build_src.py", line 153, in build_sources
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\command\build_src.py", line 286, in build_library_sources
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\command\build_src.py", line 369, in generate_sources
        File "numpy\core\setup.py", line 669, in get_mathlib_info
      RuntimeError: Broken toolchain: cannot link a simple C program

      During handling of the above exception, another exception occurred:

      Traceback (most recent call last):
        File "<string>", line 1, in <module>
        File "C:\Users\user\AppData\Local\Temp\pip-install-d360k4uy\thinc\setup.py", line 261, in <module>
          setup_package()
        File "C:\Users\user\AppData\Local\Temp\pip-install-d360k4uy\thinc\setup.py", line 201, in setup_package
          setup(
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\__init__.py", line 144, in setup
          _install_setup_requires(attrs)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\__init__.py", line 139, in _install_setup_requires
          dist.fetch_build_eggs(dist.setup_requires)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\dist.py", line 717, in fetch_build_eggs
          resolved_dists = pkg_resources.working_set.resolve(
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\pkg_resources\__init__.py", line 780, in resolve
          dist = best[req.key] = env.best_match(
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\pkg_resources\__init__.py", line 1065, in best_match
          return self.obtain(req, installer)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\pkg_resources\__init__.py", line 1077, in obtain
          return installer(requirement)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\dist.py", line 787, in fetch_build_egg
          return cmd.easy_install(req)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\command\easy_install.py", line 679, in easy_install
          return self.install_item(spec, dist.location, tmpdir, deps)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\command\easy_install.py", line 705, in install_item
          dists = self.install_eggs(spec, download, tmpdir)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\command\easy_install.py", line 890, in install_eggs
          return self.build_and_install(setup_script, setup_base)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\command\easy_install.py", line 1158, in build_and_install
          self.run_setup(setup_script, setup_base, args)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\command\easy_install.py", line 1144, in run_setup
          run_setup(setup_script, args)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\sandbox.py", line 253, in run_setup
          raise
        File "c:\users\user\appdata\local\programs\python\python38\lib\contextlib.py", line 131, in __exit__
          self.gen.throw(type, value, traceback)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\sandbox.py", line 195, in setup_context
          yield
        File "c:\users\user\appdata\local\programs\python\python38\lib\contextlib.py", line 131, in __exit__
          self.gen.throw(type, value, traceback)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\sandbox.py", line 166, in save_modules
          saved_exc.resume()
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\sandbox.py", line 141, in resume
          six.reraise(type, exc, self._tb)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\_vendor\six.py", line 685, in reraise
          raise value.with_traceback(tb)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\sandbox.py", line 154, in save_modules
          yield saved
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\sandbox.py", line 195, in setup_context
          yield
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\sandbox.py", line 250, in run_setup
          _execfile(setup_script, ns)
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\sandbox.py", line 45, in _execfile
          exec(code, globals, locals)
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\setup.py", line 443, in <module>
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\setup.py", line 435, in setup_package
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\core.py", line 171, in setup
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\__init__.py", line 145, in setup
          return distutils.core.setup(**attrs)
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\core.py", line 148, in setup
          dist.run_commands()
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\dist.py", line 966, in run_commands
          self.run_command(cmd)
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\dist.py", line 985, in run_command
          cmd_obj.run()
        File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\setuptools\command\bdist_egg.py", line 163, in run
          self.run_command("egg_info")
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\cmd.py", line 313, in run_command
          self.distribution.run_command(command)
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\dist.py", line 985, in run_command
          cmd_obj.run()
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\command\egg_info.py", line 26, in run
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\cmd.py", line 313, in run_command
          self.distribution.run_command(command)
        File "c:\users\user\appdata\local\programs\python\python38\lib\distutils\dist.py", line 985, in run_command
          cmd_obj.run()
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\command\build_src.py", line 142, in run
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\command\build_src.py", line 153, in build_sources
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\command\build_src.py", line 286, in build_library_sources
        File "C:\Users\user\AppData\Local\Temp\easy_install-wp64lity\numpy-1.17.3\numpy\distutils\command\build_src.py", line 369, in generate_sources
        File "numpy\core\setup.py", line 669, in get_mathlib_info
      RuntimeError: Broken toolchain: cannot link a simple C program
      LINK : fatal error LNK1104: cannot open file 'kernel32.lib'
      ----------------------------------------
  ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
  ----------------------------------------
ERROR: Command errored out with exit status 1: 'c:\users\user\appdata\local\programs\python\python38\python.exe' 'c:\users\user\appdata\local\programs\python\python38\lib\site-packages\pip' install --ignore-installed --no-user --prefix 'C:\Users\user\AppData\Local\Temp\pip-build-env-qwm0zsrq\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools 'wheel>0.32.0,<0.33.0' Cython 'cymem>=2.0.2,<2.1.0' 'preshed>=2.0.1,<2.1.0' 'murmurhash>=0.28.0,<1.1.0' 'thinc>=7.0.8,<7.1.0' Check the logs for full command output.


please help me solve this problem asap

--- a/chatterbot/__init__.py
+++ b/chatterbot/__init__.py
@@ -4,7 +4,7 @@ ChatterBot is a machine learning, conversational dialog engine.
 from .chatterbot import ChatBot
 
 
-__version__ = '1.2.0'
+__version__ = '1.2.1'
 
 __all__ = (
     'ChatBot',

